### PR TITLE
R docs 1536 1556 1613

### PR DESCRIPTION
**Pull request by morhilai**
A few small fixes in three pages in accordance with these issues:
https://issues.hibernatingrhinos.com/issue/RDoc-1536
https://issues.hibernatingrhinos.com/issue/RDoc-1556
https://issues.hibernatingrhinos.com/issue/RDoc-1613

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/server/extensions/revisions.markdown [hunk 1]
```diff
@@ -16,9 +16,9 @@ You can configure this for all collections (using the default configuration) and
 
 | Configuration option | Description |
 | - | - |
-| **PurgeOnDelete** | Configure whether to delete the revisions upon document delete or create a delete marker instead. |
-| **MinimumRevisionsToKeep ** | Configure how much revisions to keep. Default: unlimited. |
-| **MinimumRevisionAgeToKeep ** | Configure a minimum retention time before the revisions can be expired. Default: None. |
+| **PurgeOnDelete** | Configure whether to delete the revisions upon document delete or create a delete marker instead. Default: false. |
+| **MinimumRevisionsToKeep** | Configure the minimum number of revisions to keep. Default: none. |
```
Comment (karmeli87) [inline Documentation/4.1/Raven.Documentation.Pages/server/extensions/revisions.markdown:20]: to me `none` imply that no revision will be kept.

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown [hunk 2]
```diff
@@ -149,8 +149,8 @@ SecurityClearance: `Cluster Admin`
 
 ### Example
 
-{CODE-BLOCK:plain}
-curl -X POST http://<server-url>/admin/cluster/bootstrap
+{CODE-BLOCK: bash}
+curl -X POST http://<server-url>/admin/cluster/bootstrap -d '{}'
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:153]: should be `-d ''`

Please also apply that to other curl examples that use POST

Comment (morhilai) [reply Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:153]: Fixed (this was the only instance)

File: Documentation/4.1/Raven.Documentation.Pages/server/extensions/revisions.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/server/extensions/revisions.markdown:20]: Fixed

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown [hunk 3]
```diff
@@ -134,7 +134,7 @@ SecurityClearance: `Operator`
 
 ### Example
 
-{CODE-BLOCK:plain}
+{CODE-BLOCK: bash}
 curl -X POST http://<server-url>/admin/cluster/timeout
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:138]: add `-d ''` ?

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown [hunk 4]
```diff
@@ -120,7 +120,7 @@ SecurityClearance: `Operator`
 
 ### Example
 
-{CODE-BLOCK:plain}
+{CODE-BLOCK: bash}
 curl -X POST http://<server-url>/admin/cluster/reelect
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:124]: add `-d ''` ?

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown [hunk 5]
```diff
@@ -106,7 +106,7 @@ SecurityClearance: `Operator`
 
 ### Example
 
-{CODE-BLOCK:plain}
+{CODE-BLOCK: bash}
 curl -X POST http://<server-url>/admin/cluster/demote?nodeTag=<node-tag>
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:110]: add `-d ''` ?

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown (hunk 3 above)
Comment (morhilai) [reply Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:138]: Right - fixed

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown (hunk 4 above)
Comment (morhilai) [reply Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:124]: fixed

File: Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown (hunk 5 above)
Comment (morhilai) [reply Documentation/4.0/Raven.Documentation.Pages/server/clustering/cluster-api.markdown:110]: fixed

**Pull request merged**